### DISCUSSION TITLE
Update: June 6, 2023

### DISCUSSION THREAD

**Discussion by sebastiandig** [USF-IMARS/mbon_zooplankton_to_dwc#10] [General]
We were mostly doing a walkthrough of the zooplankton data to DarwinCore pipeline. I think pretty much everything is done related to formatting and columns needed for the zooplankton that Natalia analyzed. The only thing left is the MoF​s that don't have definitions in measurementType​. For those at the moment, I will filter out the ones with NA​ in measurementType​ to be added later as we update the MoF​ info. 

We also talked about the process to convert Anni's eDNA data. Carolina and I met yesterday to start the pipeline with what files she has. It's in the edna​ branch of mbon_zooplankton_to_dwc​. Step 1 was to convert the OTU​ and sample​ matrix into long format; step 2, join the OTU​ and taxa​ info. Both are done. Now, adding the extra information from WoRMS​ to the taxa​ info is next. I have some ideas, but I don't currently have the data to start it.

Tasks:
zooplankton:
- [x] filter `NA` in `measurementType` in the `mof_core`
- [x] save new example
- [x] update `IOOS` in our [data review](https://github.com/ioos/bio_data_guide/issues/136)

eDNA (@cperaltab)
- [x] work eDNA taxa info
- [x] get eDNA data into cloud

**Comment by 7yl4r** [comment]
My understanding is that OTUs do not translate into taxa. OTUs allow you to differentiate organisms but not identify them.

**Comment by sebastiandig** [comment]
From what Carolina had showed me. There were two documents:
1.  OTU number, column header = sample location, rows = count

| OTU  | sample1 | Sample2 | ... | sample x |
| ------ | ---------- | --------- | -- |  --------- | 
| OTU1 |       5      |   6          | ...  |       0      | 
 
2. OTU number, column header = phylogenetic tree, rows = scientific names

| OTU  |  kingdom | family | class | ...  | genus | species |
| ----- | ----------- | ------ | ----- |  -- | -------|  ---------|
| OTU1 | x              |   y       |   z     | ... |   a        |    b |

**Comment by cperaltab** [comment]
Hi all, regrading the specific tasks needed for the MBON Zooplankton dataset I can see that there are mostly done. 

1. He have agreed in using the measurementIDs and measurementTypes as we currently have it and populate the rest after, once we find the correct IDs.
2. We have solved the institutionID and collectionCode
3. LifeStage: use NERC vocab
4. Digna mentioned some issues with some dates? 

What else is missing? are we ready to send the data to Abby? I would think that yes. 

About the eDNA derived data, OTUs are considered like a kind of hypothetical taxonomic groups. In any case, I received an OTU dataset from the data provider (Anni Djurhuus) with the taxonomic groups associated to each OTU. This dataset is very close to be ready to submit to OBIS. The only detail is that I didn't receive the sequence data from the data provider. I was told that this his highly recommended but still we can send it to OBIS without it and probably submit the sequences later on. Anny suggestions about this last fact?
